_events/sr2019/competition.md

```diff
@@ -53,14 +53,14 @@ Media consent forms will also be available at reception.
 
 The competition will be held in the Students' Union building at the [University of Southampton's Highfield campus][soton-campus-directions]:
 
-Student's Union  
-University of Southampton  
-University Road  
-Southampton  
-SO17 1BJ  
-United Kingdom  
-
-<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2514.4224296392836!2d-1.399433684090786!3d50.934400360256014!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x487473f7089148ab%3A0xaeeea75e9ec72c88!2sUniversity+of+Southampton+Students&#39;+Union+(SUSU)!5e0!3m2!1sen!2suk!4v1551272133022" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
+Student's Union
+University of Southampton
+University Road
+Southampton
+SO17 1BJ
+United Kingdom
+
```

Comment:
This whitespace change is going to be breaking -- the trailing whitespace here is important to mark that these lines should be wrapped as they are.